style.css

```diff
@@ -0,0 +1,717 @@
+    padding-left: 34px;
+    padding-right: 15px;
+    z-index: 3;
+}
+.catalog__title::before {
+    content: "";
+    width: 24px;
+    height: 24px;
+    margin-right: 8px;
+    background-image: url('img/folder.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+    position: absolute;
+    top: 50%;
+    left: 0;
+    /* margin-top: -12px; */
+    transform: translateY(-50%);
+    z-index: 2;
+}
+.catalog__buttons {
+    background-color: #0D0D15;
+    position: relative;
+    z-index: 3;
+    display: flex;
+    padding-left: 15px;
+}
+.catalog__buttons a {
+    position: relative;
+}
+.catalog__buttons a::after {
+    content: "";
+    position: absolute;
+    top: 10px;
+    right: -12px;
+    z-index: 2;
+    background-color: rgba(208, 215, 251, 0.17);
+    width: 2px;
+    height: 24px;
+}
+.catalog__buttons a + button {
+    margin-left: 20px;
+}
+.catalog__buttons button + button {
+    margin-left: 10px;
+}
+.btn—prev,
+.btn—next {
+    background-color: transparent;
+    text-decoration: none;
+    color: #F5F8FB; 
+    border-width: 2px;
+    border-style: solid;
+    border-color: rgba(208, 215, 251, 0.17);
+    height: 48px;
+    width: 48px;
+    align-items: center;
+    justify-content: center;
+}
+.btn—prev:hover,
+.btn—next:hover {
+    background-color: #0265DC;
+}
+.btn—prev::after {
+    content: "<";
+}
+.btn—next::after {
+    content: ">";
+}
+.catalog__list {
+    gap: 10px;
+    display: flex;
+}
+.catalog__item {
+    width: 300px;
+    background-color: #1D1C28;
+    padding: 16px 22px 16px 16px;
+}
+.catalog__image img {
+    width: 100%;
+    height: auto;
+}
+.catalog__image{
+    position: relative;
+}
+.catalog__label {
+    position: absolute;
+    top: 8px;
+    right: 3px;
+    background-color: #F5F8FB;
+    color: #0265DC;
+    font-size: 14px;
+    font-weight: 600;
+    padding: 4px 8px;
+    text-transform: uppercase;
+}    
+.catalog__label--red {
+    background-color: #EA3829;
+    color: #F5F8FB;
+}
+.catalog__price {
+    display: block;
+    font-family: "Bakbak One", Times New Roman, Georgia, serif;
+    font-size: 36px;
+    line-height: 48px;
+}
+.catalog__heading {
+    display: block;
+    color: #EFF5FA;
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+}
+.section-title {
+    text-align: center;
+    font-weight: 400;
+    color: #F8F7FF;
+    margin-bottom: 25px;
+}
+.blog {
+    margin: 0 auto;
+    max-width: 1758px;
+    padding: 0 15px;
+    margin-bottom: 100px;
+}
+.blog__list {
+    display: flex;
+    gap: 10px;
+    flex-wrap: wrap;
+    margin-bottom: 40px;
+}
+.blog__item {
+    background-color: #1D1C28;
+    padding: 16px;
+    flex-basis: calc(50% - 10px);
+    flex-shrink: 1;
+    color: rgba(239, 245, 250, 0.82);
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+}
+.blog__img {
+    position: relative;
+    margin-bottom: 15px;
+}
+.blog__img img {
+    max-width: 100%;
+    height: auto;
```

Comment:
I think in a lecture about graphics, these styles will have to be told that they can be made global for all img